### ISSUE TITLE
polypolish_insert_filter.py is missing

### ISSUE THREAD

**Issue by schlaepferp** [rrwick/Perfect-bacterial-genome-tutorial#12]
Dear Ryan,

Similar as in the other issue, polypolish_insert_filter.py seems to be missing?
This time I can't find the python script.
Would it be possible to provide it?

Cheers,
Pascal

**Comment by schlaepferp** [comment]
I just realized this:
https://github.com/rrwick/Polypolish/releases
It states:
The insert-size filtering step used to be implemented in a separate Python script named polypolish_insert_filter.py. I have now implemented it in Rust with the rest of Polypolish. This is faster and Polypolish now exists as a single executable file with two subcommands:
Previously: polypolish_insert_filter.py ..., currently: polypolish filter ...
Previously: polypolish ..., currently: polypolish polish ...
So maybe modify this and update the command in the easy part?
